pkg/util/helper/patch_test.go

```diff
@@ -88,7 +88,7 @@ func TestGenMergePatch(t *testing.T) {
 			patch, err := GenMergePatch(testObj, tc.modifyFunc())
 			if err != nil && tc.expectErr == false {
 				t.Fatalf("unexpect error, but got: %v", err)
-			} else if err == nil && tc.expectErr == true {
+			} else if err == nil && tc.expectErr {
```

Comment:
This modification is unnecessary.

Reply:
Ok, thanks. I'll close this pr

Reply:
In any case, still thank you for your contribution!